### ISSUE TITLE
Setting tabindex

### ISSUE THREAD

**Issue by BottomleyIan** [mmomtchev/rlayers#236]
Is it currently possible to set the tab index used by the map div. I could not see anything. If set it stops the mouse zoom interaction until the map has focus

https://openlayers.org/en/latest/examples/accessible.html

Thanks

**Comment by mmomtchev** [comment]
No, but it is trivial to add, I will merge a PR if you do it, you can use `minZoom`/`maxZoom` as examples

**Comment by BottomleyIan** [comment]
Created PR https://github.com/mmomtchev/rlayers/pull/237
